en/08_Changelogs/5.4.0.md

```diff
@@ -45,6 +46,18 @@ The changes include:
 - Reports list is now paginated
 - [`Description`](api:SilverStripe\Reports\Report::description()) is now displayed in the list as a column
 
+### New `class_description` configuration on `DataObject` {#class-description}
+
+[`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) and [`BaseElement`](api:DNADesign\Elemental\Models\BaseElement) both seperately implemented a `description` configuration property which was used to describe the purpose of a given subclass. For `SiteTree` this is used when creating a new page in the CMS. For `BaseElement` this is used in the elemental blocks report. In both cases the purpose is the same - it provides additional context about the intended use case for a given subclass.
+
+We've now implemented this concept in `DataObject` directly with the new [`DataObject.class_description`](api:SilverStripe\ORM\DataObject->class_description) configuration property. It is now considered best practice to add a description of `DataObject` subclasses using this configuration. You can use the new [`DataObject::classDescription()`](api:SilverStripe\ORM\DataObject::classDescription()) and [`DataObject::i18n_classDescription()`](api:SilverStripe\ORM\DataObject::i18n_classDescription()) methods if you need a description of any `DataObject` class. Those methods already existed on `SiteTree` but have been moved up the hierarchy so they can be called on any `DataObject` class.
+
+For now this is only used in the same places that used to use the deprecated configuration, but future minor releases are likely to broaden the scope of its usage.
```

Comment:
I could see it being used as a tooltip when hovering over elemental blocks in the "Add new block" list and added to the `GridFieldAddNewMultiClass` component in gridfield extensions at the very least.